broker/src/test/java/io/camunda/zeebe/broker/bootstrap/ClusterServicesStepTest.java

```diff
@@ -50,14 +53,15 @@ void shouldCompleteFutureOnStartup() {
     sut.startupInternal(mockBrokerStartupContext, CONCURRENCY_CONTROL, future);
 
     // then
-    assertThat(future.isDone()).isTrue();
-    assertThat(future.isCompletedExceptionally()).isFalse();
+    assertThat(future).succeedsWithin(TIME_OUT);
```

Comment:
:wrench: You can access the result after a `succeedsWithin`, e.g. `assertThat(future).succeedsWithin(TIME_OUT).isNotNull()`. I use [this IntelliJ extension](https://plugins.jetbrains.com/plugin/12195-concise-assertj-optimizing-nitpicker-cajon-) to learn these things as it suggests to me to combine assertions, or use more explicit ones. It's not always perfect, but it's generally pretty good.